modules/nw-cluster-mtu-change.adoc

```diff
@@ -0,0 +1,309 @@
+--
+
+... To update the `MTU` field in the configuration file created in the previous step, enter the following command:
++
+[source,terminal]
+----
+$ sed '/^MTU/s/=.*$/=<mtu>/' ifcfg.txt
+----
++
+--
+where:
+
+`<mtu>`:: Specifies the hardware MTU for the primary interface.
+--
+
+... To base64 encode the NetworkManager configuration for inclusion in a MachineConfig object, enter the following command:
++
+[source,terminal]
+----
+cat ifcfg.txt | base64 -w0
+----
+
+... Create two MachineConfig objects for the control plane nodes and the worker nodes in your cluster:
+
+.... Create the following MachineConfig object in the `machine-config-control-plane.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 01-control-plane-interface
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<encoding_config>
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/sysconfig/network-scripts/ifcfg-<connection_name>
+    systemd: {}
+----
++
+--
+where:
+
+`<encoding_config>`:: Specifies the base64 encoding for the updated NetworkManager configuration.
+`<connection_name>`:: Specifies the NetworkManager connection name for the primary network interface.
+--
+
+.... Create the following MachineConfig object in the `machine-config-worker.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 01-master-interface
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<encoding_config>
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/sysconfig/network-scripts/ifcfg-<connection_name>
+    systemd: {}
+----
++
+--
+where:
+
+`<encoding_config>`:: Specifies the base64 encoding for the updated NetworkManager configuration.
+`<connection_name>`:: Specifies the NetworkManager connection name for the primary network interface.
+--
+
+** If your hardware MTU is specified with DHCP, update your DHCP configuration such as with the following dnsmasq configuration:
++
+[source,text]
+----
+dhcp-option-force=26,<mtu>
+----
++
+--
+where:
+
+`<mtu>`:: Specifies the hardware MTU for the DHCP server to advertise.
+--
+
+** If your hardware MTU is specified with a kernel command line with PXE, update that configuration accordingly.
```

Comment:
Is it actually possible to change the kernel command line after deployment?

Otherwise, NM profiles would be generated from them, and the previous instruction on how to update the NM profile would apply. With the caveat that the `connection.autoconnect-priority` would have to be set to a value above the default of 0.

Reply:
@jcaamano, interesting; Not sure, I'm open to suggestions about the best way(s) to advise users to modifying the underlying MTU. This is mostly based on an unpublished KCS that I found for how to do this.

Reply:
No idea myself. Will let other chime in on this one.